client/modules/IDE/components/Searchbar/Searchbar.jsx

```diff
@@ -26,11 +26,12 @@ class Searchbar extends React.Component {
 
   handleSearchEnter = (e) => {
     if (e.key === 'Enter') {
-      this.props.setSearchTerm(this.state.searchValue);
+      this.searchChange();
     }
   }
 
-  searchChange = (value) => {
+  searchChange = () => {
+    if (this.state.searchValue.trim().length === 0) return;
     this.props.setSearchTerm(this.state.searchValue);
```

Comment:
Shouldn't you be using `this.state.searchValue.trim()` as the input value here?

Reply:
yes, will do the recommended changes